alfresco-health-processor-platform/src/main/java/eu/xenit/alfresco/healthprocessor/plugins/solr/SolrRequestExecutor.java

```diff
@@ -46,7 +47,7 @@ public SolrSearchResult checkNodeIndexed(SearchEndpoint endpoint, Collection<Sta
         // Initially, try a fetch for double the size of the node statuses array
         // This is so we can immediately detect the case where all nodes are indexed twice.
         int fetchSize = nodeStatuses.size() * 2;
-        JsonNode response = executeSearchRequest(endpoint, nodeStatuses, fetchSize);
+        JsonNode response = executeSearchRequest(endpoint, nodeStatuses, fetchSize, checkTransaction);
```

Comment:
Why pass this as a parameter to the `executeSearchRequest` method?

The method is on the same object, so you can switch based on the `checkTransaction` property there?

Reply:
This is indeed not needed